mobile/build.gradle

```diff
@@ -122,6 +123,17 @@ def readProguardFiles() {
     return proguardFiles
 }
 
+tasks.withType(Checkstyle) {
+    configFile = rootProject.file('config/checkstyle/checkstyle.xml')
```

Comment:
@pg301 When it is at `config/checkstyle/checkstyle.xml` then it is not needed to be specified. That is the magic of convention. But creating multiple folders for such minimal things will also be wrong way. So please move them to `misc/` :wink: